src/HBitcoin.Tests/FullSpvWalletTests.cs

```diff
@@ -158,14 +147,13 @@ public void SycingTest(string networkString)
 		
 		[Fact]
 		public void HaveFundsTest()
-		{
-			// load wallet
-			Network network = Network.TestNet;
-			string path = $"CommittedWallets/HaveFunds{network}.json";
-			const string password = "";
-			Safe safe = Safe.Load(password, path);
-			Assert.Equal(safe.Network, network);
-			System.Diagnostics.Debug.WriteLine($"Unique Safe ID: {safe.UniqueId}");
+		{		
+            // load wallet
+            Network network = Network.TestNet;
+            Safe safe = Safe.Load(string.Empty, $"../../CommittedWallets/HaveFunds{network}.json");
```

Comment:
../../ is used to navigate to the folder CommittedWallets.
Otherwise I get an exception when running this test.